src/applications/vaos/utils/data.js

```diff
@@ -19,17 +19,6 @@ export function arrayToObject(items, idProp = 'id') {
   );
 }
 
-/**
```

Comment:
@vbahinwillit FYI I figured I'll remove these generic utilities since they seem to have been added for specific components before that have since been removed. I figure if we need them, we can easily find the code in the commit history. There's really not much benefit to keep the dead code around but let me know if you prefer that I don't remove these!